cmd/cmd_controller.js

```diff
@@ -404,10 +402,27 @@ class EmbarkController {
     var fs = require('../lib/core/fs.js');
     fs.removeSync('./chains.json');
     fs.removeSync('.embark/');
+    fs.removeSync('node_modules/.cache');
     fs.removeSync('dist/');
     console.log(__("reset done!").green);
   }
 
+  ejectWebpack() {
+    var fs = require('../lib/core/fs.js');
+    var dappConfig = fs.dappPath('webpack.config.js');
+    var embarkConfig = fs.embarkPath('lib/pipeline', 'webpack.config.js');
+    if (fs.existsSync(dappConfig)) {
```

Comment:
there is also the option of moving the old one to `webpack.config.js.old`
It changes a bit the user experience, what do you think?

Reply:
We could definitely do that, but then what if `old` already exists? Create `old.1`, and so on?

Reply:
I was looking here for example: https://github.com/facebook/create-react-app/blob/next/packages/react-scripts/scripts/init.js#L236
They don't seem to even bother, just display a warning
But yes that would work

Reply:
Done.